Doc/Tutorial/chapter_uniprot.tex

```diff
@@ -25,15 +25,15 @@ \subsection{Parsing Swiss-Prot records}
 >>> from urllib.request import urlopen # Python 3 only
 >>> from io import TextIOWrapper
 >>> 
->>> handle = TextIOWrapper(urlopen("http://example.org/myprositefile.dat"))
+>>> handle = TextIOWrapper(urlopen("https://raw.githubusercontent.com/biopython/biopython/master/Tests/SwissProt/F2CXE6.txt"))
```

Comment:
There is also another issue with the file that you mentioned (since they were my first choice):

```python
handle = TextIOWrapper(urlopen("https://raw.githubusercontent.com/biopython/biopython/master/Tests/Prosite/ps00107.txt"))
>>> record = SwissProt.read(handle)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/gasta/.local/lib/python3.5/site-packages/Bio/SwissProt/__init__.py", line 145, in read
    record = _read(handle)
  File "/home/gasta/.local/lib/python3.5/site-packages/Bio/SwissProt/__init__.py", line 182, in _read
    _read_id(record, line)
  File "/home/gasta/.local/lib/python3.5/site-packages/Bio/SwissProt/__init__.py", line 294, in _read_id
    raise ValueError("ID line has unrecognised format:\n" + line)
ValueError: ID line has unrecognised format:
ID   PROTEIN_KINASE_ATP; PATTERN.
```
Probably a different module from `Bio.SwissProt` must be used.